src/components/DataTable/DataTable-story.js

```diff
@@ -0,0 +1,580 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import classNames from 'classnames';
+import DataTableRow from '../DataTableRow';
+import DataTableData from '../DataTableData';
+import DataTableRowExpand from '../DataTableRowExpand'; // eslint-disable-line
+import {
+  DataTableExpandableRow, // eslint-disable-line
+  DataTableExpandableRowContent, // eslint-disable-line
+} from '../DataTableExpandableRow';
+import {
+  DataTable,
+  DataTableToolbar,
+  DataTableToolbarContent,
+  DataTableToolbarAction,
+  DataTableHead,
```

Comment:
Yeah, that makes sense! I named it `DataTableHead` because of the HTML element `thead`. So I might keep `DataTableHead` but then change `DataTableHeader` to `DataTableColumnHeader`.

Reply:
Makes sense.